src/plugins/telemetry/telemetry_impl.cpp

```diff
@@ -1061,6 +1068,31 @@ void TelemetryImpl::process_sys_status(const mavlink_message_t& message)
     }
 }
 
+void TelemetryImpl::process_battery_status(const mavlink_message_t& message)
+{
+    mavlink_battery_status_t bat_status;
+    mavlink_msg_battery_status_decode(&message, &bat_status);
+
+    _has_bat_status = true;
+
+    Telemetry::Battery new_battery;
+    new_battery.id = bat_status.id;
+    new_battery.voltage_v = bat_status.voltages[0] * 1e-3f;
+    // FIXME: it is strange calling it percent when the range goes from 0 to 1.
```

Comment:
Good point :(

Reply:
I actually copied this block from the status message, but indeed, it is a good point :)

Reply:
Ah nevermind then.